File: server/makc2022--dotnet/Makc2022.Layer5.Sql.Server/Pages/DummyMain/Item/Queries/Get/DummyMainItemPageGetQueryOutput.cs
// Copyright (c) 2022 Maxim Kuzmin. All rights reserved. Licensed under the MIT License.

using DummyMainDomainItemGetQueryOutput = Makc2022.Layer4.Sql.Domains.DummyMain.Queries.Item.Get.DomainItemGetQueryOutput;

namespace Makc2022.Layer5.Sql.Server.Pages.DummyMain.Item.Queries.Get
{
    /// <summary>
    /// Выходные данные запроса на получение страницы сущности "DummyMain".
    /// </summary>
    public class DummyMainItemPageGetQueryOutput
    {
        #region Properties

        /// <summary>
        /// Выходные данные запроса на получение элемента в домене "DummyMain".
        /// </summary>
        public DummyMainDomainItemGetQueryOutput? OutputOfDummyMainDomainItemGetQuery { get; set; }

        #endregion Properties
    }
}

File: server/makc2022--nodejs--nestjs/src/layer5/server/pages/dummy-main/item/queries/get/dummy-main-item-page-get-query-output.ts
/** Copyright (c) 2022 Maxim Kuzmin. All rights reserved. Licensed under the MIT License. */

import { DomainItemGetQueryOutput as DummyMainDomainItemGetQueryOutput } from '../../../../../../../layer4/domains/dummy-main/queries/item/get/domain-item-get-query-output';

/** Выходные данные запроса на получение страницы сущности "DummyMain". */
export class DummyMainItemPageGetQueryOutput {
  /** Выходные данные запроса на получение элемента в домене "DummyMain". */
  outputOfDummyMainDomainItemGetQuery = new DummyMainDomainItemGetQueryOutput();
}
